show/main.py

```diff
@@ -2464,6 +2464,9 @@ def summary():
 
     manager = HealthCheckerManager()
     chassis = Chassis()
+    if not manager.config.config_file_exists():
```

Comment:
Could we move this line before "chassis = Chassis()"

Reply:
fixed